flows/contact.go

```diff
@@ -196,14 +196,14 @@ type fieldValueEnvelope struct {
 }
 
 type contactEnvelope struct {
-	UUID        ContactUUID                     `json:"uuid" validate:"required,uuid4"`
-	Name        string                          `json:"name"`
-	Language    utils.Language                  `json:"language"`
-	Timezone    string                          `json:"timezone"`
-	URNs        URNList                         `json:"urns"`
-	GroupUUIDs  []GroupUUID                     `json:"group_uuids,omitempty" validate:"dive,uuid4"`
-	Fields      map[FieldKey]fieldValueEnvelope `json:"fields,omitempty"`
-	ChannelUUID ChannelUUID                     `json:"channel_uuid,omitempty" validate:"omitempty,uuid4"`
+	UUID     ContactUUID                     `json:"uuid" validate:"required,uuid4"`
+	Name     string                          `json:"name"`
+	Language utils.Language                  `json:"language"`
+	Timezone string                          `json:"timezone"`
+	URNs     URNList                         `json:"urns"`
+	Groups   []*GroupReference               `json:"groups,omitempty" validate:"dive"`
+	Fields   map[FieldKey]fieldValueEnvelope `json:"fields,omitempty"`
+	Channel  *ChannelReference               `json:"channel,omitempty" validate:"omitempty,dive"`
```

Comment:
possibly these shouldn't be pointers, but if not that's for another day as they are pointers everywhere else